ocs_ci/helpers/keyrotation_helper.py

```diff
@@ -408,3 +408,120 @@ def wait_till_keyrotation(self, device_handle):
             assert False
 
         return True
+
+
+def enable_key_rotation():
```

Comment:
I did not understand this function, keyrotation is enabled by default. what are we enabling here?

Reply:
Key rotation is enabled while deploying the cluster. That too its a choice, not mandatory. Even it is enabled while deploying the cluster, there is an option to disable it. So, we are just checking whether it can be enabled or not before starting our testing. If the key rotation didn't select in the cluster deployment, the test will fail here itself. I am just following the steps what feature test case followed.